src/platform/macos/window.rs

```diff
@@ -582,6 +583,12 @@ impl WindowExt for Window2 {
     fn get_nsview(&self) -> *mut c_void {
         *self.view as *mut c_void
     }
+
+    #[inline]
+    fn set_blur_material(&self, material: BlurMaterial) {
+        let view = self.get_nsview() as *mut objc::runtime::Object;
+        unsafe { msg_send![view, setMaterial: material as i64]; }
```

Comment:
In every other usage, we call `msg_send!` on `*self.view`. Doing that is much cleaner (and has less potential overhead) than calling `get_nsview` like this.

Additionally, with `setMaterial: material`, from what I've seen Obj-C style has us writing `setMaterial:material`. This applies to the other occurrences as well, naturally.